instat/dlgView.vb

```diff
@@ -174,7 +174,11 @@ Public Class dlgView
                 ucrBase.OKEnabled(False)
             End If
         Else
-            ucrBase.OKEnabled(True)
+            If ucrSelectorForView.ucrAvailableDataFrames.cboAvailableDataFrames.Text <> "" Then
+                ucrBase.OKEnabled(True)
+            Else
+                ucrBase.OKEnabled(False)

```

Comment:
Minor fix here. I realised if you open the dialog without any data loaded then you can press OK still